implementations/rust/kex/src/x3dh.rs

```diff
@@ -329,7 +334,7 @@ impl KeyExchanger for X3dhResponder {
                     h: state_hash,
                     encrypt_key,
                     decrypt_key,
-                    local_static_secret: self.identity_key,
+                    local_static_secret: self.identity_key.unwrap(),
```

Comment:
does this need to be handled rather than unwrapping?

Reply:
Same as previous

Reply:
If this is true because it's "been checked" prior, then shouldn't we use a new variable at the point you prove it's `Some` and use that elsewhere?

Reply:
No because it has to be passed across function calls which are not directly linked.